woocommerce/woocommerce-gutenberg-products-block
# Create CountryInput component

Part of #1574.

For now, I made it use countries directly printed in the settings. But as discussed in #1574 and in preparation for #1575 we would like to get the list of countries/states from the API. But I feel we can work on that in a follow-up PR, while keeping this centered around the component & style.

#### Accessibility




### Screenshots

![imatge](https://user-images.githubusercontent.com/3616980/72525747-27466a80-3865-11ea-9fd9-6984d2d54fa1.png)

### How to test the changes in this Pull Request:
1. Create a post with a _Checkout_ block and modify [this line](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/assets/js/blocks/cart-checkout/checkout/block.js#L112) so input fields appear:
```diff
-{ shippingMethods.length > 0 && (
+{ true && (
```
2. Verify the Country input is a select and matches the styles.
